dspace-server-webapp/src/main/resources/application.properties

```diff
@@ -38,6 +38,11 @@
 # interact with or read its configuration from dspace.cfg.
 dspace.dir=${dspace.dir}
 
+########################
+# Servlet context path configuration for spring boot application running with embedded tomcat
+#
+server.servlet.context-path=/server
```

Comment:
but this is from v8, right?!

Reply:
It is in v8, but we had that in previous version of the customer/uk.